openapi.yaml

```diff
@@ -197,16 +201,98 @@ paths:
         name: limit
         schema:
           type: integer
+      - in: query
+        name: offered_by
+        schema:
+          type: array
+          items:
+            enum:
+            - mitx
+            - ocw
+            - bootcamps
+            - xpro
+            - csail
+            - professional education
+            - sloan executive education
+            - schwarzman college of computing
+            - center for transportation & logistics
+            type: string
+            description: |-
+              * `mitx` - mitx
+              * `ocw` - ocw
+              * `bootcamps` - bootcamps
+              * `xpro` - xpro
+              * `csail` - csail
+              * `professional education` - professional education
+              * `sloan executive education` - sloan executive education
+              * `schwarzman college of computing` - schwarzman college of computing
+              * `center for transportation & logistics` - center for transportation & logistics
       - in: query
         name: offset
         schema:
           type: integer
+      - in: query
+        name: platform
+        schema:
+          type: array
+          items:
+            enum:
+            - edx
+            - ocw
+            - open learning library
+            - mitx online
+            - bootcamps
+            - xpro
+            - csail
+            - professional education
+            - sloan executive education
+            - schwarzman college of computing
+            - center for transportation & logistics
+            - whu
+            - susskind
+            - global alumni
+            - simplilearn
+            - emeritus
+            - podcast
+            type: string
+            description: |-
+              * `edx` - edx
+              * `ocw` - ocw
+              * `open learning library` - open learning library
```

Comment:
Do we want this to be `"oll"`, etc? If yes, is that in-scope for this PR?

Reply:
Do we want it to be OLL in the results returned or just the filter? In any case that's probably a separate pr